libs/sdk-backend-tiger/src/backend/workspace/execution/executionResult.ts

```diff
@@ -126,6 +162,29 @@ export class TigerExecutionResult implements IExecutionResult {
             return new TigerDataView(this, result, this.dateFormatter);
         });
     };
+
+    private handleExportResultPolling(
+        client: ITigerClient,
+        payload: ActionsApiGetTabularExportRequest,
+    ): Promise<IExportResult> {
+        return client.export
+            .getTabularExport(payload, {
+                transformResponse: (x) => x,
+            })
+            .then((result) => {
+                if (result?.status === 200) {
+                    return {
+                        uri: result?.config?.url || "",
+                    };
+                } else {
+                    return new Promise((resolve) => {
+                        setTimeout(() => {
+                            resolve(this.handleExportResultPolling(client, payload));
```

Comment:
This will probably consume all the errors, we should likely propagate them upper in the Promise chain?

Reply:
Fixed, used async, adopted similar polling options to bear, confirmed with exporters